dvc/commands/experiments/ls.py

```diff
@@ -18,15 +18,13 @@ def run(self):
         )
         tags = self.repo.scm.describe(exps)
         remained = {baseline for baseline, tag in tags.items() if tag is None}
-        base = "refs/heads/"
+        base = "refs/heads"
         ref_heads = self.repo.scm.describe(remained, base=base)
```

Comment:
@karajan1001, can we push this inside `experiments.ls()`,

Reply:
agreed, ideally it should be part of the lower level API ... if there are no particular reason to keep it on the level of command, I can take care of this in a follow up PR.

Let's fix the bug first though and release it. The whole command is broken, it's not good to make ppl wait for this.

Reply:
I can easily imagine we'll want these details (proper names) in `--json` in VS Code ... and probably we already do something like this in `exp show`.

Reply:
yeah, sorry, blocking the PR was not my intention. Just noting down what I noticed.

Reply:
yep, nw, good catch @skshetry!

Reply:
Yes, I agreed. Should we make this change a new PR?